templates/publisher/listing.html

```diff
@@ -9,17 +9,17 @@
     {% set selected_tab='listing' %}
     {% include "publisher/_header.html" %}
 
-    <div class="p-strip is-shallow">
+    <div class="p-strip is-shallow u-no-padding--top">
         <form id="market-form" method="POST" enctype='multipart/form-data'>
           <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
           <input type="hidden" name="snap_id" value="{{ snap_id }}" />
           <input type="file" name="icon" id="snap_icon" hidden class="hidden" accept="image/*"/>
 
-          <div class="row">
+          <div class="row snapcraft-p-sticky">
             <div class="col-12 u-align--right">
```

Comment:
Can we include the Snap title on the Left hand side (desktop) or Top (mobile)?

Reply:
There's going to be a message about linking to the 'snap listings' or other such in the next PR I do (after this one is merged) https://github.com/CanonicalLtd/snapcraft-design/issues/405